common.gypi

```diff
@@ -407,9 +407,15 @@
         'libraries': [ '-lelf' ],
       }],
       ['OS=="freebsd"', {
-        # Use this flag because on FreeBSD std::pairs copy constructor is non-trivial
-        # https://lists.freebsd.org/pipermail/freebsd-toolchain/2016-March/002094.html
-        'cflags': [ '-D_LIBCPP_TRIVIAL_PAIR_COPY_CTOR=1' ],
+        'conditions': [
+          ['llvm_version < "4.0"', {
```

Comment:
Ah, in that case don't worry about it. We'll have to update everything anyway when 10.0 comes.